atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -537,7 +560,20 @@ private void refreshTokenRanges() {
     private <V, K extends Exception> V runOnHost(InetSocketAddress specifiedHost,
                                                  FunctionCheckedException<Cassandra.Client, V, K> fn) throws K {
         CassandraClientPoolingContainer hostPool = currentPools.get(specifiedHost);
-        return hostPool.runWithPooledResource(fn);
+        totalRequests.getAndIncrement();
```

Comment:
Can we have requests metric per container too? That will involve currentPools.forEach((address, cassandraClientPoolingContainer) ->
               registerMetric(..))